docs/json_rpc_api.mediawiki

```diff
@@ -2094,9 +2094,9 @@ of the best block.
 |-
 !Parameters
 |
-# <code>command</code>: <code>(string, required)</code>  <code>connect</code> to add a peer (defaults to temporary), <code>remove</code> to remove a persistent peer, or <code>disconnect</code> to remove all matching non-persistent peers.
-# <code>peer</code>: <code>(string, required)</code> ip address and port, or ID of the peer to operate on.
-# <code>connection type</code>: <code>(string)</code> <code>perm</code> indicates the peer should be added as a permanent peer, <code>temp</code> indicates a connection should only be attempted once.
+# <code>subcmd</code>: <code>(string, required)</code> <code>disconnect</code> to remove all matching non-persistent peers, <code>remove</code> to remove a persistent peer, or <code>connect</code> to connect to a peer.
+# <code>target</code>: <code>(string, required)</code> Either the IP address and port of the peer to operate on, or a valid peer ID.
+# <code>connectsubcmd</code>: <code>(string)</code> <code>perm</code> to make the connected peer a permanent one, <code>temp</code> to try a single connect to a peer.
```

Comment:
I don't see anything wrong with this change, but I did want to point out that the parameters are all positional, so they really don't need to match the names in the structs if that is the goal.

Perhaps if we ever add support for named parameters (which is supported by the JSON-RPC 2.0 spec and means it would need to be limited accordingly, because JSON-RPC 1.0 requires positional parameters), it would make sense to ensure the names match.

Reply:
Good point. The parameter names in the docs match the names in the structs for pretty much all of the other handlers, so I think it makes sense to keep them the same for consistency.